rust/stackable-cockpit/src/platform/service.rs

```diff
@@ -45,15 +45,54 @@ pub async fn get_endpoints(
     object_name: &str,
     object_namespace: &str,
 ) -> Result<IndexMap<String, String>, Error> {
-    let service_list_params =
+    let mut endpoints = IndexMap::new();
```

Comment:
note: Why do we create a `IndexMap` here, even if we could fail at line 55. I would suggest to move the instantiation back below the K8s client list call.

Reply:
No real reason, changed in 4c7b9df